scripts/scan-g-cloud-services-for-bad-words.py

```diff
@@ -115,17 +119,31 @@ def check_services_with_bad_words(output_dir, framework_slug, client, suppliers,
                         for word in bad_words:
                             if get_bad_words_in_value(word, service.get(key)):
                                 output_bad_words(
-                                    supplier["supplierId"], framework_slug, service["id"],
-                                    service["serviceName"], service["serviceSummary"], key,
-                                    service.get(key), word, writer)
+                                    supplier["supplierId"],
+                                    framework_slug,
+                                    service["id"],
+                                    service["serviceName"],
+                                    service.get("serviceSummary", service["serviceDescription"]),
```

Comment:
It's happy with G9 because G9 uses serviceDescription, so direct access works. If you run it against G8 or earlier it will break.